Dnn.CommunityForums/Deprecated/API.cs

```diff
@@ -65,7 +65,7 @@ public int Forums_Save(int PortalId, DotNetNuke.Modules.ActiveForums.Forum fi, b
 			try
 			{
 				var fc = new ForumController();
-				return fc.Forums_Save(PortalId, fi, isNew, UseGroup);
+				return fc.Forums_Save(PortalId, fi, isNew, UseGroup, UseGroup);
```

Comment:
@johnhenley This appears to be using the same variable twice... but one is supposed to reference a "security" variable, right?

Reply:
this is intentional; it's for backward compatibility and the method is unused, and being deprecated.